apps/web/urls.py

```diff
@@ -1,6 +1,6 @@
 from django.urls import path
 
-from .views import Contact, Events, Home, News, Products
+from .views import Contact, Events, Home, News, Products, About
```

Comment:
```suggestion
from .views import About, Contact, Events, Home, News, Products
```

Quando formos passar teste de qualidade de código ele vai gritar se não estiver em ordem alfabética xD (boas práticas).

Reply:
EU achei que fosse na ordem que aparece no arquivo! Ótimo saber disso! :}